articles/expressroute/expressroute-troubleshooting-arp-classic.md

```diff
@@ -110,7 +110,7 @@ The ARP table of a peering can be used to validate Layer 2 configuration and con
 * The ARP table has an entry for the on-premises side with a valid IP and MAC address, and a similar entry for the Microsoft side.
 * The last octet of the on-premises IP address is always an odd number.
 * The last octet of the Microsoft IP address is always an even number.
-* The same MAC address appears on the Microsoft side for all three peerings (primary/secondary).
+* The same MAC address appears on the Microsoft side for all three peerings (Azure private, Azure public, and Microsoft).
```

Comment:
Please remove Azure public as this is no longer offered.

Reply:
@duongau - should this wait until the change you requested has been made?